src/inet/tests/BUILD.gn

```diff
@@ -23,19 +23,12 @@ config("tests_config") {
   include_dirs = [ "." ]
 }
 
-chip_test_suite("tests") {
-  output_name = "libInetLayerTests"
-
+static_library("chip_inet_test_common_source") {
```

Comment:
I think this will break nRF connect & ESP32 tests.

Shouldn't need to do change this, the tests are already a static library, see below comment.